ext/fg/js/popup.js

```diff
@@ -160,7 +154,7 @@ class Popup {
     }
 
     isVisibleSync() {
-        return this._isInjected && (this._visibleOverride !== null ? this._visibleOverride : this._visible);
+        return (this._visibleOverride !== null ? this._visibleOverride : this._visible);
```

Comment:
This should work because `_visible` won't be true before injection has completed

Reply:
Tested that screenshot still works